tests/qunit/index.html

```diff
@@ -2,10 +2,62 @@
 	<span class="button-text">{{{ settings.text }}}</span>
 </script>
 
+
 <script type="text/html" id="tmpl-elementor-tag-post-custom-field-content">
 	<span>{{{  }}}</span>
 </script>
 
+
```

Comment:
Try to use the original templates, maybe via a grunt task that extracts them before the test.

Reply:
We cannot yet, its better to have a new task for handling the whole problem for `navigator` As example we need `elementor-editor-wrapper` which created in PHP, the same is also for `elementor-container` which causes issue with `resizeColumn` test